hex/sd_api_v2/connectivity_2.0.1_115k2_with_s130_2.0.1.hex

```diff
@@ -8059,7 +8059,7 @@
 :10FFE0000A460146682003F0E5FB002810BD0E20FC
 :10FFF00010BD4058002800D00120704701000007C4
 :020000040002F8
-:1000000017A5D84602FFFFFF00000000020000FF16
+:1000000017A5D84602FFFFFF00000000020001FF15
 :100010000201FFFF00C20100FFB581B014460D00D0
 :10002000074623D0002C21D013460A4601466920FA
 :10003000266803F0BFFB002802D1002F02D0002069
```

Comment:
Just noticed: Only one line (version number I assume) has changed in this file, but two lines have changed in the other ones. Is this as expected?

Reply:
It's expected because a timeout value was changed for all hex files except one. Other than that the version number was changed in all 4 files.